Packs/TaniumThreatResponse/ReleaseNotes/2_0_3.md

```diff
@@ -0,0 +1,17 @@
+
+#### Integrations
+##### Tanium Threat Response v2
+- Added the ability to filter command '**tanium-tr-list-connections**' response according to status/ip/platform/hostname/id.
+- Added the ability to filter command '**tanium-tr-get-system-status**' response according to status/ip_client/ip_server/port/hostname.
+- Added the ability to filter command '**tanium-tr-event-evidence-list**' response according to hostname/type.
+- Added the ability to filter command '**tanium-tr-list-file-downloads**' response according to hash/hostname/process_time_start.
+
+#### Playbooks
+#### Tanium Threat Response - Create Connection
+- - Update playbook **playbook-Tanium Threat Response - Create Connection** description
+#### Tanium Threat Response - Request File Download
+- Update playbook **playbook-Tanium Threat Response - Request File Download** description
+#### Tanium Threat Response - Create Connection v2
+- Add playbook **playbook-Tanium Threat Response V2 - Create Connection** to correspond TaniumThreatResponseV2 Integration.
+##### Tanium Threat Response - Request File Download v2
+- Add playbook **playbook-Tanium Threat Response V2 - Request File Download** to correspond TaniumThreatResponseV2 Integration.
```

Comment:
put here the playbook desc
also, follow the `New` entity release notes convention

Reply:
not sure where i can find those release notes conventions? in developer docs?

Reply:
I managed to do that simply by re-running the generate docs of the sdk